db/upgrade/2.5.1-upgrade-2.6.0-mysql.sql

```diff
@@ -0,0 +1,18 @@
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements.  See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership.  The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- this file works for MySQL.
+ALTER TABLE `shenyu`.`plugin` ADD COLUMN `plugin_class` mediumblob NULL COMMENT 'plugin jar class';
```

Comment:
you should add it to MySQL/Oracle/PostgreSQL, not only mysql.

Reply:
yes,I wil do it.